src/components/ui/navigation-menu.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import * as NavigationMenuPrimitive from "@radix-ui/react-navigation-menu";
```

Comment:
Don't know if this could also be split into multiple imports @kalisowski could you check that out?